dll/win32/shell32/lang/en-US.rc

```diff
@@ -980,6 +980,11 @@ BEGIN
     IDS_OBJECTS "%d Objects"
     IDS_OBJECTS_SELECTED "%d Objects Selected"
 
+    IDS_TITLE_MYCOMP "My Computer"
```

Comment:
Where do these translations come from?

Reply:
Translation comes from existing strings (`IDS_MYCOMPUTER`, `IDS_NETWORKPLACE`, `IDS_RECYCLEBIN_FOLDER_NAME`).

Reply:
In the same file? :thinking: then what about using those existing string IDs?

Reply:
Windows has separate resources for these specific things, I'm following it :slightly_smiling_face:

Reply:
ah, so it duplicates the strings too? SUS but OK....